core/evancore.vue.libs/src/components/forms/form/form.vue

```diff
@@ -68,13 +68,13 @@
                 :is="getControlComponentName(controlName)"
                 :id="form[controlName].uiSpecs && form[controlName].uiSpecs.attr ? form[controlName].uiSpecs.attr.id : null"
                 v-model="form[controlName].value"
-                :disabled="!editable || isLoading || form[controlName].uiSpecs.prohibited === true"
+                :disabled="!editable || isLoading || (form[controlName].uiSpecs && form[controlName].uiSpecs.prohibited === true)"
```

Comment:
Can we refactor this into a `isDisbaled` getter? Would extract the logic from template and make it less verbose. Could consider the same for `isProhibited`.